static/src/stylesheets/module/content/_content.scss

```diff
@@ -407,7 +407,7 @@
     }
 
     @include mq(leftCol) {
-        min-height: gs-height(1) + $gs-baseline;
```

Comment:
Hey, just noticed that the dateline min-height had gone awry and tracked it down to this line, do you mind fixing?